src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -79,25 +81,36 @@ public void doGet(
 
     // Get tasks from Google Tasks
     TasksClient tasksClient = tasksClientFactory.getTasksClient(googleCredential);
-    List<TaskList> taskLists = tasksClient.listTaskLists();
-    List<Task> tasks = getTasks(tasksClient);
+    List<TaskList> allTaskLists = tasksClient.listTaskLists();
 
     // Initialize Tasks Response
-    List<String> taskListTitles = getTaskListTitles(taskLists);
+    Map<String, String> allTaskListIdsAndTitles = getTaskListIdsAndTitles(allTaskLists);
```

Comment:
Nit: `allTaskListIdsToTitles` would make the map contents a bit more clear (here and everywhere else)

Reply:
I agree, changed and thanks!

Reply:
`RESOLVED`